src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/ActivityListener.cs

```diff
@@ -47,6 +47,11 @@ public ActivityListener()
         /// </summary>
         public GetRequestedData<ActivityContext>? GetRequestedDataUsingContext { get; set; }
 
+        /// <summary>
+        /// Determine if the listener automatically generates a new trace Id before sampling when there is no parent context.
```

Comment:
Could you make it explicit that this traceid will be used by the Activity, should we chose to create it?